NOTICE

```diff
@@ -0,0 +1,25 @@
+FlaskBB includes code adapted from other Python libaries or resources, including:
```

Comment:
Miguel probably wouldn't come after us for my spin on his soft deletes, however since I've harped on other packages for not doing this properly, I feel better with this here.

Reply:
yeah, lets give credit where credit is due :)